package/src/components/Message/Message.tsx

```diff
@@ -561,6 +570,20 @@ const MessageWithContext = <
 
   const { userLanguage } = useTranslationContext();
 
+  // TODO: Can be removed in V6 and from here completely once it becomes baseline.
+  const PollWrapper = useCallback(() => {
```

Comment:
We need it so that we can render the preview of the `Poll` message on long press of the message itself. Instead of passing a bunch of props (so that overrides are respected) we can pass a component wrapper that knows how to render itself from the downstream.

Reply:
Cool